### PR TITLE
Add map_normalize Presto function

### PR DESCRIPTION
**Pull request by mbasmanova**
Differential Revision: D54918766

### CODE REVIEW

File: velox/functions/prestosql/MapNormalize.h [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    // semantics.
+
+    for (const auto& entry : inputMap) {
+      if (!entry.second.has_value()) {
+        auto& keyWriter = out.add_null();
+        keyWriter.copy_from(entry.first);
+      } else {
+        auto [keyWriter, valueWriter] = out.add_item();
+        keyWriter.copy_from(entry.first);
+        valueWriter = entry.second.value() / totalSum;
```
Comment (xiaoxmeng) [inline velox/functions/prestosql/MapNormalize.h:48]: There is no exception throw for double zero?

**Review by xiaoxmeng — APPROVED** [review]
@mbasmanova LGTM. Thanks!

File: velox/functions/prestosql/MapNormalize.h (hunk 1 above)
Comment (mbasmanova) [reply velox/functions/prestosql/MapNormalize.h:48]: No. The test covers this case.

**Review by amitkdutta — APPROVED** [review]
Thanks @mbasmanova Looks good.

**Pull request closed without merging**